app/src/main/java/io/mvpstarter/sample/data/remote/ApiManager.java

```diff
@@ -1,26 +1,29 @@
-package io.mvpstarter.sample.data;
+package io.mvpstarter.sample.data.remote;
 
 import java.util.List;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
 import io.mvpstarter.sample.data.model.Pokemon;
-import io.mvpstarter.sample.data.remote.MvpStarterService;
 import io.reactivex.Single;
 
+/**
+ * Created by shivam on 29/5/17.
+ */
+
 @Singleton
-public class DataManager {
```

Comment:
@ShivamDev31 This cannot be ApiManager, Please revert. Idea of DataManager is to handle everything on the data layer, not just API.